test/hotspot/jtreg/runtime/valhalla/inlinetypes/JumboInline.java

```diff
@@ -44,56 +44,27 @@
     final long l18;
     final long l19;
 
-    private JumboInline() {
-        l0 = 0;
-        l1 = 0;
-        l2 = 0;
-        l3 = 0;
-        l4 = 0;
-        l5 = 0;
-        l6 = 0;
-        l7 = 0;
-        l8 = 0;
-        l9 = 0;
-        l10 = 0;
-        l11 = 0;
-        l12 = 0;
-        l13 = 0;
-        l14 = 0;
-        l15 = 0;
-        l16 = 0;
-        l17 = 0;
-        l18 = 0;
-        l19 = 0;
-    }
-
-    public static JumboInline createJumboInline() {
-        JumboInline j = JumboInline.default;
-        return j;
-    }
-
-    public JumboInline update(long l0, long l1) {
-        JumboInline j = __WithField(this.l0, l0);
-        j = __WithField(j.l1, l1);
-        j = __WithField(j.l2, l0 + l1);
-        j = __WithField(j.l3, l1 + l2);
```

Comment:
Bug here (and the following lines): `l2` refers to `this.l2`, not `j.l2`—if `this` is `JumboInline.default`, that value is 0, not `l0+l1`.

Reply:
Agreed